.env.production

```diff
@@ -0,0 +1,4 @@
+ADDRESS_SERVER_URL=https://beta.3box.io/address-server
+ORBITDB_PATH=/opt/orbitdb
+#IPFS_PATH=/opt/ipfs
```

Comment:
This shouldn't be commented out right?

Reply:
It was in the current code, can't remember why, would have to look back to confirm why. Was set null before this PR

Reply:
It was like that because we used the default and I didn't know what that was. We should use the config now instead.
@c-castillo what path should we set for ipfs?

Reply:
Or rather, what path are we using right now?

Reply:
we are using `/opt/ipfs` since we are symlinking the default one `$HOME/.ipfs` to it

Reply:
Ok, great! Then we can just remove the `#`!

Reply:
And add the config to the ipfs node also :)